# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/fping/fping.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur -x .deps -x Makefile fping-2.4b2/Makefile.am compiled-fping/Makefile.am
--- fping-2.4b2/Makefile.am	Thu Mar 15 03:07:52 2001
+++ compiled-fping/Makefile.am	Wed Feb 19 11:55:59 2003
@@ -1,5 +1,5 @@
+AUTOMAKE_OPTIONS = 1.3 foreign
 
 sbin_PROGRAMS = fping
 fping_SOURCES = fping.c options.h linux.h
 man_MANS = fping.8
-AUTOMAKE_OPTIONS=foreign
diff -Nur -x .deps -x Makefile fping-2.4b2/configure.in compiled-fping/configure.in
--- fping-2.4b2/configure.in	Thu Mar 15 03:09:54 2001
+++ compiled-fping/configure.in	Wed Feb 19 11:58:44 2003
@@ -1,6 +1,7 @@
 dnl Process this file with autoconf to produce a configure script.
-AC_INIT(fping.c)
-AC_CONFIG_HEADER(config.h)
+AC_INIT
+AM_INIT_AUTOMAKE(fping,2.4b1)
+AM_CONFIG_HEADER(config.h)
 VERSION=2.4b1
 PACKAGE=fping
 AC_SUBST(PACKAGE)
diff -Nur -x .deps -x Makefile fping-2.4b2/fping.c compiled-fping/fping.c
--- fping-2.4b2/fping.c	Fri Jul 20 19:10:26 2001
+++ compiled-fping/fping.c	Wed Feb 19 11:53:27 2003
@@ -65,7 +65,7 @@
 
 /*** autoconf includes ***/
 
-#include <config.h>
+#include "config.h"
 
 #include <stdio.h>
 #include <errno.h>
